examples/basic_secondary_ranges/main.tf

```diff
@@ -14,16 +14,12 @@
  * limitations under the License.
  */
 
+provider "google" {
```

Comment:
I believe this approach is deprecated, please switch back to `required_providers`.

Reply:
Gotcha: there was a mix of the two approaches in the existing examples. I'd assumed one was intended for modules and one for projects, but I'll switch them all to use `required_providers`.

Reply:
I'm going to move the required providers into the `versions.tf` file where it exists.

Reply:
Also, I remember this problem from my contribution to one of the other models: I can't retrieve the Cloud Build logs, although I can see the `int` and `lint` triggers have failed. If you can paste any info here so I know what to look at/fix that'd be good.